<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>661</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>132</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="plotWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="comboBox_ports"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_baudrate">
           <property name="toolTip">
            <string>Выберите скорость передачи данных</string>
           </property>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000000</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_connect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Строка для передачи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_send">
           <property name="text">
            <string>Send cmd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_saveCSV">
           <property name="text">
            <string>Сохранить в CSV</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
